Sources/StytchCore/StytchClient/OAuth/OAuth+Apple.swift

```diff
@@ -59,6 +59,21 @@ public extension StytchClient.OAuth {
 
             return authenticateResponse
         }
+
+        // sourcery: AsyncVariants, (NOTE: - must use /// doc comment styling)
+        /// This function is intended for rare cases where you must authenticate with Apple only and collect the JWT.
+        /// This function will not create a Stytch user.
+        public func authenticateWithApple() async throws -> String {
```

Comment:
nit: naming. I don't love it since it describes what it does internally, not what a developer is using it for/what it returns. Maybe something like `getAppleIdToken` or something like that? But it's your call!